### DISCUSSION TITLE
How to get mesh plots to work

### DISCUSSION THREAD

**Discussion by gwsteffens-az** [phoebe-project/phoebe1#17] [Q&A]
I've successfully modeled an over contact not in thermal contact eclipsing binary system. The LC plots work great showing the observed and calculated LC. However, every time I try to display the Mesh Plot, the Phoebe program crashes.  I've successfully duplicated the parameters in WD2015 and LC with MPAGE=5 generates the data for plotting the surface elements of the binary system. 
I realize Phoebe Legacy isn't supported any more and I hope to migrate to Phoebe2 some day, but it would be great if you can offer any guidance on how to get the mesh plots to work.
Thanks!
Gary

**Comment by aprsa** [comment]
Hi Gary,

Can you please attach the parameter file (no need to include any data files)? Also, which version of legacy and what OS are you using?

Cheers,
Andrej

**Reply by gwsteffens-az** [reply]
Hi Andrej, Thank you for your quick reply. I'm using 0.32-snapshot on Windows 10. I've attached my parameter file. 
Thanks! Gary
[My-Binary.txt](https://github.com/phoebe-project/phoebe1/files/8347184/My-Binary.txt)
